unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendsHUD/Scripts/Tabs/FriendsTabViewBase.cs

```diff
@@ -198,17 +231,12 @@ protected virtual void OnPressBlockButton(string userId, bool blockUser)
         }
     }
 
-    public virtual void CreateOrUpdateEntry(string userId, FriendEntryBase.Model model)
-    {
-        CreateEntry(userId);
-        UpdateEntry(userId, model);
-    }
-
-    public virtual bool CreateEntry(string userId)
+    protected virtual bool CreateEntry(string userId)
     {
         if (entries.ContainsKey(userId)) return false;
 
         PoolableObject newFriendEntry = friendEntriesPool.Get();
+        newFriendEntry.gameObject.name = $"FriendEntry {userId}";
```

Comment:
We should be wrapping this line under UNITY_EDITOR to save browser time performance.

Reply:
That should be gone, it was just debugging stuff I forgot. Going to remove it!